src/book/mod.rs

```diff
@@ -221,7 +221,14 @@ impl MDBook {
 
             debug!("[*]: Writing to .gitignore");
 
+            // Write the destination folder to file
             writeln!(f, "{}", relative).expect("Could not write to file.");
+            
+            // Add common files
+            let ignores = vec!("*~", ".DS_Store");
```

Comment:
Hi @dns2utf8 thanks for the submission.

I'm not convinced that we should try to add "common" (or any for that matter) ignore patterns here. We would not like to start catering to requests for adding everyone's favorite temp file formats as this is trivially handled by [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore). This `.gitignore` serves a specific purpose and it cannot be easily handled via global ignore patterns (actually you can build a whole hierarchy of gitignores).

Could you give a usecase that cannot be handled by global patterns?.